### DISCUSSION TITLE
Locations where CondaPkg stores settings

### DISCUSSION THREAD

**Discussion by jtcantin** [JuliaPy/CondaPkg.jl#125] [Q&A] [answered]
Hello,

I am not able to get CondaPkg to forget about a pip package that was installed, but that I no longer want installed and no longer exists on pypi.

I changed the CondaPkg.toml file, deleted .CondaPkg, deleted Manifest.toml, deleted all pycache files and re-ran `instantiate` (after closing and reopening the REPL). 

However, whenever I try to use my local package, CondaPkg still tries to install the old package (and thus errors out). 

I have also tried `gc --all` in the Julia pkg REPL, but this does not fix the problem.

I am working on Ubuntu 20.04 on WSL2 and using VSCode.

Where else can I find a setting or lurking environment folder that is causing CondaPkg to think it should install the old package?

Thanks

**Comment by jtcantin** [comment, marked as answer]
I got it fixed. After completely uninstalling and reinstalling Julia, I got the attached error. I then restarted my computer and everything worked.
I guess there was some old process that did not die when killing the Julia terminal that was inserting old settings.

![Screenshot 2024-02-28 141757_2](https://github.com/JuliaPy/CondaPkg.jl/assets/4580993/61413510-81ad-4499-9d7e-53e69e448b8b)

**Reply by cjdoris** [reply]
I'm glad you fixed your problem! It sounds like something buggy on your system and not an issue with CondaPkg itself.

For the record, the only files that CondaPkg depends on are:
- Any CondaPkg.toml files it finds in any packages in your load path. These are all printed to screen when CondaPkg is resolving dependencies.
- The .CondaPkg directory in the active project. This contains the actual Conda environment plus a file of metadata about the resolved environment.

So deleting .CondaPkg and checking all the CondaPkg.toml files do not include the offending package should have been enough.
